### PR TITLE
Add text extraction example

### PR DESCRIPTION
**Pull request by bootstraponline**

### CODE REVIEW

**Comment by bootstraponline** [comment]
When doing video player testing, this technique is required since we want to keep track of the current position labels.

File: docs/faq.md [hunk 1]
```diff
@@ -51,6 +51,29 @@ following:
...
+ *
+ *  var textValue:String = ""
+ *
+ *  domainField.performAction(grey_replaceText("hello.there"))
+ *             .performAction(grey_getText(&textValue))
+ */
+func grey_getText(inout text: String) -> GREYActionBlock {
+    return GREYActionBlock.actionWithName("get text",
+                                           constraints: grey_respondsToSelector(Selector("text")),
+                                          performBlock: { element, errorOrNil -> Bool in
```
Comment (khandpur) [inline docs/faq.md]: is this formatted correctly?

File: docs/faq.md [hunk 2]
```diff
@@ -51,6 +51,29 @@ following:
...
+Here's an example of storing an element's text attribute.
+
+```swift
+// Swift
+/*
+ *  Example Usage:
+ *
+ *  var textValue:String = ""
+ *
+ *  domainField.performAction(grey_replaceText("hello.there"))
```
Comment (khandpur) [inline docs/faq.md]: grey_replaceText isn't checked in yet :)

Comment (bootstraponline) [inline docs/faq.md]: oh, oops.

File: docs/faq.md (hunk 1 above)
Comment (bootstraponline) [inline docs/faq.md]: I'm not actually sure how to format this correctly. I made another attempt. Is there a Swift style guide?

**Comment by bootstraponline** [comment]
I think the new formatting is an improvement, let me know.

File: docs/faq.md [hunk 3]
```diff
@@ -51,6 +51,29 @@ following:
...
+// Swift
+/*
+ *  Example Usage:
+ *
+ *  var textValue:String = ""
+ *
+ *  domainField.performAction(grey_typeText("hello.there"))
+ *             .performAction(grey_getText(&textValue))
+ */
+func grey_getText(inout text: String) -> GREYActionBlock {
```
Comment (khandpur) [inline docs/faq.md]: This is style guide we can follow until google publishes one:

https://github.com/raywenderlich/swift-style-guide

Comment (bootstraponline) [inline docs/faq.md]: Cool, we should probably add that to the contributing.md file. I pushed a new update.

Comment (khandpur) [inline docs/faq.md]: Good point. Opened an issue:
https://github.com/google/EarlGrey/issues/131

**Pull request merged**